src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/partition.py

```diff
@@ -114,9 +114,28 @@ def capture_gptq_patterns(self, graph_module: GraphModule):
                     for pattern_op in enabled_ops:
                         self.supported_ops.enable_by_name(pattern_op)
 
+    def capture_nncf_patterns(self, graph_module: GraphModule):
+        const_node = PatternNode
+        const_node.op_types["get_attr"] = None
+        bitwise_right_shift_node = PatternNode
+        bitwise_right_shift_node.op_types["call_function:aten.bitwise_right_shift.Tensor_Scalar"] = [const_node]
+        bitwise_and_node = PatternNode
+        bitwise_and_node.op_types["call_function:aten.bitwise_and.Scalar"] = [const_node,]
+        stack_node = PatternNode
+        stack_node.op_types["call_function:aten.stack.default"] = [bitwise_and_node, bitwise_right_shift_node]
+
+        for node in graph_module.graph.nodes:
+            if str(node.op) == "call_function" and str(node.target) == "aten.stack.default":
+                enabled_ops = []
+                pattern_match = self.check_pattern(node, bitwise_and_node, enabled_ops)
+                if pattern_match:
+                    for pattern_op in enabled_ops:
+                        self.supported_ops.enable_by_name(pattern_op)
+
     def make_partitions(self, graph_module: GraphModule, options) -> GraphModule:
         allow_single_node_partition = _is_testing(options)
         self.capture_gptq_patterns(graph_module)
+        self.capture_nncf_patterns(graph_module)
```

Comment:
Has this been tested with the torch.compile openvino backend option 'aot_autograd':True, which is required for dynamic shape support?

Reply:
Thanks for the comment. I have checked and the pattern is valid.